code/src/sixsq/nuvla/server/resources/nuvlabox_peripheral.clj

```diff
@@ -129,13 +129,14 @@ nuvlabox.
 ;;
 
 (defmethod crud/add-acl resource-type
-  [resource _request]
-  (when-let [nuvlabox-id (:parent resource)]
+  [resource request]
+  (if-let [nuvlabox-id (:parent resource)]
```

Comment:
Parent should be mandatory for nuvlabox peripheral. Does it make sense to have nb periph not attached to a nuvlabox ?

Reply:
it is but the schema validation only happens after this